docs/contents/releases/v1-18-eks-1.md

```diff
@@ -0,0 +1,641 @@
+# EKS-D v1-18-eks-1 Version Depedencies
+
+The EKS Distro of Kubernetes source repository does not include any AMIs (Amazon
```

Comment:
This text is great, it might make sense to move this to a different 'releases' page so we don't ahve to include it on every release page?

Reply:
Done.